test/k8sT/Policies.go

```diff
@@ -381,22 +385,17 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 
 			By("Testing if egress policy enforcement is enabled on the endpoint")
 
-			var epList []models.Endpoint
-			err = kubectl.CiliumEndpointsList(ciliumPod).Unmarshal(&epList)
-			Expect(err).To(BeNil(), "Getting a list of endpoints from %s", ciliumPod)
-
-			epsWithEgress := 0
-			for _, ep := range epList {
-				for _, lbls := range ep.Status.Labels.SecurityRelevant {
-					if lbls == "k8s:io.kubernetes.pod.namespace="+helpers.DefaultNamespace {
-						switch ep.Status.Policy.Realized.PolicyEnabled {
-						case models.EndpointPolicyEnabledBoth, models.EndpointPolicyEnabledEgress:
-							epsWithEgress++
-						}
-					}
-				}
+			Expect(kubectl.WaitCEPReady()).To(BeNil(), "Cep is not ready in a timeout")
```

Comment:
Nit: in a timeout --> after a  specified timeout

Change this for all CEP-related timeout error description messages in this commit.